runtime/compiler/env/J9PersistentInfo.hpp

```diff
@@ -452,6 +458,8 @@ class PersistentInfo : public OMR::PersistentInfoConnector
    bool        _localSyncCompiles;
    bool        _JITServerUseAOTCache;
    std::string _JITServerAOTCacheName; // Name of the server AOT cache that this client is using
+   bool        _JITServerUseAOTCachePersistence; // Whether to persist the JITServer AOT caches at the server
```

Comment:
Nitpick: we should try to keep the fields' order consistent with their initialization order in the constructor.